ansible_collections/arista/cvp/plugins/modules/cv_configlet.py

```diff
@@ -683,6 +685,9 @@ def main():
     if not tools.HAS_DIFFLIB:
         module.fail_json(msg='difflib required for this module')
 
+    if not tools.HAS_HASHLIB:
```

Comment:
This block should also be part of cv_configlet_v3.py file

Reply:
Added imports in tools_cv.py and checks in cv_configlet_v3.py. Since both hashlib and difflib are built-in modules, it's hard to test the module failure is working correctly. :-)